src/test/java/microsoft/exchange/webservices/data/core/request/UserConfigurationTest.java

```diff
@@ -0,0 +1,41 @@
+package microsoft.exchange.webservices.data.core.request;
```

Comment:
please add the [license header](https://github.com/OfficeDev/ews-java-api/blob/master/license.txt) to all new files.